maestro-client/src/main/java/maestro/ScrollDirection.kt

```diff
@@ -0,0 +1,15 @@
+package maestro
+
+enum class ScrollDirection {
```

Comment:
Just a though, what if we call those: `ABOVE`, `BELOW`, `RIGHT`, `LEFT`? To avoid confusion with up/down scrolling

Reply:
This is indeed pretty confusing

Reply:
> ABOVE, BELOW, RIGHT, LEFT

This seems more confusing.

It is Up/Down scrolling, however it's the opposite of swipe direction. i.e `Swipe->UP` is the same as `Scroll->Down`

To me it makes sense that "Scroll Down" means it will start scrolling towards the bottom. Open to revise if users find it confusing.

Reply:
What about naming them:

```
FROM_CENTER_LEFT_TO_CENTER_RIGHT
FROM_CENTER_RIGHT_TO_CENTER_LEFT
FROM_CENTER_TOP_TO_CENTER_BOTTOM
FROM_CENTER_BOTTOM_TO_CENTER_TOP
```

Reply:
Thanks @edgarfroes. We'll keep it as is for now and re-evaluate if necessary.